VKappProject/VKappProject/Controllers/GroupsTableViewController.swift

```diff
@@ -0,0 +1,56 @@
+// GroupsTableViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Экран групп
+final class GroupsTableViewController: UITableViewController {
+    // MARK: - Private Properties
+
+    private var myGroups = [groups.first].compactMap { $0 } {
+        didSet {
+            tableView.reloadData()
+        }
+    }
+
+    // MARK: - Public Methods
+
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        guard segue.identifier == Constants.Identifiers.addGroupIdentifier,
+              let availableTableVC = segue.destination as? AvailableTableViewController else { return }
+        availableTableVC.addGroup(myGroups) { [weak self] group in
+            self?.myGroups.append(group)
+        }
+    }
+
+    // MARK: - UITableViewDelegate, UITableViewDataSource
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
```

Comment:
```suggestion
    override func numberOfSections(in tableView: UITableView) -> Int {
```
Если нет разделения по секциям можно не писать